py/occampy2/wrappers/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .model import Model

```

Comment:
I don't think there is a need in these for python 2, as well as all other classes from wrappers. We discussed to leave their old stuff unmodified and work only on new things which will be in python 3.

Reply:
True, but I figured it can't hurt to have it there for the time being, as it was a straight copy from the python3 stuff.